app/assets/stylesheets/retro.scss

```diff
@@ -0,0 +1,68 @@
+.nav {
+  background-color: #696969;
+  height: 60px;
+}
+
+.headerItems {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.header-icon {
+  border-radius: 30px;
+  background-color: white;
+  padding: 4px;
+  margin-right: 15px;
+  margin-top: 6px;
+}
+
+#time {
```

Comment:
I think it's better to use classes instead of ids, but if there's a reason I'm good.